docs/topics/Threads.md

```diff
@@ -163,24 +163,42 @@ Additionally, when a client loses access to a channel they are not removed from
 
 Discord's clients only load active threads into memory on start. So when a thread is unarchived, there is no guarantee that the client has either the thread or whether they are a member, in memory. As such, the Gateway sends a [Thread Update](#DOCS_TOPICS_GATEWAY_EVENTS/thread-update) first, which contains the full channel object. And then sends a [Thread Member Update](#DOCS_TOPICS_GATEWAY_EVENTS/thread-member-update) to each member of the thread, so those clients know they are a member, and what their notification setting is. This event is not that valuable for bots right now, but is going to be received by bots, so is documented here none the less.
 
-# Forums
+## Forums
 
-A `GUILD_FORUM` channel is very similar (from an API perspective) to a `GUILD_TEXT` channel, except only threads can be created in that channel; messages cannot be sent directly in that channel.
+A `GUILD_FORUM` (type `15`) channel is similar to a `GUILD_TEXT` channel, except *only* threads can be created in them. Unless otherwise noted, threads in forum channels behave in the same way as in text channels—meaning they use the same endpoints and receive the same Gateway events.
```

Comment:
```suggestion
A `GUILD_FORUM` (type `15`) channel is similar to a `GUILD_TEXT` channel, with the exception that *only* threads can be created in them. Unless otherwise noted, threads in forum channels behave in the same way as in text channels, meaning they use the same endpoints and receive the same Gateway events.
```